src/driver/op.rs

```diff
@@ -123,18 +123,6 @@ where
             })
         })
     }
-
```

Comment:
Every Op\<T> had come with a *try_submit_with* but the only one that used it was the *close* op? That's interesting.